internal/client/valve/replay.go

```diff
@@ -6,15 +6,17 @@ import (
 	"io"
 	"net/http"
 	"os"
-	"strings"
+	"path"
+	"path/filepath"
 )
 
-var (
-	ErrDownloadRelpay = fmt.Errorf("failed to download the dota2 replay")
+const (
+	DefaultDestination           = "/tmp"
+	DefaultReplayExtensionSuffix = ".dem.bz2"
```

Comment:
feels like this belongs here instead of a separate util dir

Reply:
Yuup2, i agree with this